README.md

```diff
@@ -169,20 +168,18 @@ async def build_and_send():
     Generate a bundle with two private transactions and send it.
     :return:
     """
-    tx1 = '<raw_tx1>'
-    tx2 = '<raw_tx2>'
+    user_tx_hash = '<user_tx_hash>'
```

Comment:
would be great to have real signed txs here

Reply:
perhaps introduce the event stream first, then show an example using the SSE stream event in a bundle

Reply:
Sure, will add the hash + signed tx pair and the SSE/send_bundle combination.